### PR TITLE
Step 6

### PR DESCRIPTION
**Pull request by CarnevaleMNO**
## What
**Step 6**

1. Added docker compose
2. finished step 6

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  web:
+    build:
+      context: ./typescript/simple-mercari-web
+      dockerfile: dockerfile
+    container_name: build2022_web
+    ports:
+      - "3000:3000"
+    volumes:
+      - "./typescript/simple-mercari-web:/code"
+      - "./typescript/simple-mercari-web/node_modules:/code/node_modules"
```
Comment (westcottian) [inline docker-compose.yml:22]: Nice!

File: python/db.py [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+# #     id INTEGER PRIMARY KEY,
+# #     name TEXT
+# # )
+# # """)
+
+# # commit changes
+# connection.commit()
+
+# # close connection
+# connection.close()
```
Comment (westcottian) [inline python/db.py]: Better to remove these commented codes.

File: python/db.py [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+
+# add an item to category table
+# connection = sqlite3.connect("../db/mercari.sqlite3")
+# cursor = connection.cursor()
+# cursor.execute("""
+# INSERT INTO category (name)
+# VALUES (?)
+# """, ("practical",))
+# connection.commit()
+# connection.close()
```
Comment (westcottian) [inline python/db.py]: Commented code can be removed.

File: python/dockerfile [hunk 4]
```diff
@@ -1,4 +1,22 @@
-FROM alpine
+# docker build -t build2022/app:latest -f python/dockerfile .
+# docker run -d -p 9000:9000 build2022/app:latest
+# docker-compose up
+# docker-compose down
+# docker-compose down --rmi all -v # to remove all volumes and images
```
Comment (westcottian) [inline python/dockerfile]: Better we removed these lines.

File: python/dockerfile [hunk 5]
```diff
@@ -1,4 +1,22 @@
...
+WORKDIR /code
+
+COPY python/requirements.txt /code/requirements.txt
+
+RUN pip3 install --no-cache-dir --upgrade -r /code/requirements.txt
+
+COPY python/main.py /code/main.py
+COPY python/db.py /code/db.py
+COPY python/images /code/images
+COPY python/image /code/image
```
Comment (westcottian) [inline python/dockerfile:14]: Can we copy these 4 files in one statement?

File: python/dockerfile (hunk 4 above)
Comment (CarnevaleMNO) [reply python/dockerfile]: Thank you! I will remove all comments now

**Pull request merged**